networking_cisco/plugins/cisco/cfg_agent/device_drivers/asr1k/aci_asr1k_routing_driver.py

```diff
@@ -196,11 +196,8 @@ def _add_ha_hsrp(self, ri, port, is_external=False):
         group = port_ha_info['group']
         ip = self._get_interface_ip_from_hosting_port(ri, port,
             is_external=is_external)
-        if is_external:
-            ha_ip = port_ha_info['ha_port']['fixed_ips'][0]['ip_address']
```

Comment:
It seems to me we need to get this ha_ip from somewhere instead of just always use ip+1 value......